scm-ui/ui-webapp/src/containers/InitializationAdminAccountStep.tsx

```diff
@@ -0,0 +1,227 @@
+    setPassword(newPassword);
+  };
+
+  const validateAndSetPasswordConfirmation = (newPasswordConfirmation: string) => {
+    setPasswordConfirmationValidationError(newPasswordConfirmation !== password);
+    setPasswordConfirmation(newPasswordConfirmation);
+  };
+
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    create({
+      startupToken: startupKey,
+      userName,
+      displayName,
+      email,
+      password,
+      passwordConfirmation,
+    });
+  };
+
+  let errorComponent;
+  if (error) {
+    if (error.message === "Forbidden") {
+      errorComponent = <ErrorNotification error={new Error(t("error.forbidden"))} />;
+    } else {
+      errorComponent = <ErrorNotification error={error} />;
+    }
+  }
+
+  const formValid =
+    !!(startupKey && userName && displayName && email && password && passwordConfirmation) &&
+    !(
+      userNameValidationError ||
+      displayNameValidationError ||
+      mailValidationError ||
+      passwordValidationError ||
+      passwordConfirmationValidationError
+    );
+  const component = (
+    <div className="column is-8 box  has-background-white-ter">
+      <form onSubmit={handleSubmit}>
+        <h3 className="title">{t("title")}</h3>
+        <h4 className="subtitle">{t("adminStep.title")}</h4>
+        {errorComponent}
+        <p>{t("adminStep.description")}</p>
+        <div className={"columns"}>
+          <div className="column is-full-width">
+            <InputField placeholder={t("adminStep.startupToken")} autofocus={true} onChange={setStartupKey} />
+          </div>
+        </div>
+        <div className={"columns"}>
+          <div className="column is-half">
+            <InputField
+              testId="username-input"
+              label={t("adminStep.username")}
+              onChange={validateAndSetUserName}
+              validationError={userNameValidationError}
+              value={userName}
+            />
+          </div>
+          <div className="column is-half">
+            <InputField
+              testId="displayname-input"
+              label={t("adminStep.displayname")}
+              onChange={validateAndSetDisplayName}
+              value={displayName}
+              validationError={displayNameValidationError}
+            />
+          </div>
+        </div>
+        <div className={"columns"}>
+          <div className="column is-full-width">
+            <InputField
+              label={t("adminStep.email")}
+              onChange={validateAndSetEmail}
+              value={email}
+              validationError={mailValidationError}
+            />
+          </div>
+        </div>
+        <div className={"columns"}>
+          <div className="column is-half">
+            <InputField
+              testId="password-input"
+              label={t("adminStep.password")}
+              type="password"
+              onChange={validateAndSetPassword}
```

Comment:
Your password validation doesn't work as expected. I inserted "password" into both fields and afterwards changed the first field to like "123456". The form was valid again and on submit i got the backend error for "passwords doesn't match".